learn/wme-backup-restore/backup/platform/backup-aws.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Backup on AWS"
+id: ""
+sidebar_label: "Backup on AWS"
+---
+---
+
+
+Stop WME  EC2 instance and detach network interface [follow the steps given here](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html#detach_eni), make a note of interface ID or eni ID.
```

Comment:
Backup and Restore come under one file.
with two different types as I explained in the ESxi sheet